src/Server.ts

```diff
@@ -71,8 +71,8 @@ export default class Socks5Server {
     }
 
     // Not private because someone may want to inject a duplex stream to be handled as a connection
-    _handleConnection(socket: Duplex) {
+    _handleConnection(socket: net.Socket) {
```

Comment:
I want to continue supporting Duplex streams as a substitute to sockets for situations where socks5 streams need to be processed but are not received in the form of a raw socks5 tcp connection.